modules/otel-config-instrumentation.adoc

```diff
@@ -98,9 +98,46 @@ spec:
 
 |===
 
+[id="otel-configuration-of-opentelemetry-sdk-variables_{context}"]
+== Configuration of the OpenTelemetry SDK variables
+
+You can use the `instrumentation.opentelemetry.io/inject-sdk` annotation in the OpenTelemetry Collector custom resource to instruct the {OTELName} Operator to inject the following OpenTelemetry SDK environment variables into your pod:
```

Comment:
to inject some of the following OpenTelemetry SDK environment variables depending on the Instrumentation CR into your pod.


The reason being not all of these env vars are injected, e.g. if TLS is not specified the  OTEL_EXPORTER_OTLP_CERTIFICATE (and others) are not set

Reply:
Thank you. Applied.